korangar/src/graphics/passes/point_shadow/entity.rs

```diff
@@ -23,9 +23,17 @@ const INITIAL_INSTRUCTION_SIZE: usize = 256;
 #[repr(C)]
 struct InstanceData {
     world: [[f32; 4]; 4],
+    texture_top_left: [f32; 2],
+    texture_bottom_left: [f32; 2],
+    texture_top_right: [f32; 2],
+    texture_bottom_right: [f32; 2],
     texture_position: [f32; 2],
     texture_size: [f32; 2],
     depth_offset: f32,
+    depth_extra: f32,
+    angle: f32,
+    foo: f32,
```

Comment:
If you need to use padding, then just use a padding as the last field.

```
padding: [u32;2]
```

You can then initialize this with Default::default()!

Reply:
I tested it works really well.